en/config/actuators.md

```diff
@@ -79,6 +79,34 @@ The `X`, `Y`, `Z` positions are in [FRD coordinate frame, relative to the _centr
 Note, this may not be the same as the position of the flight controller!
 :::
 
+#### Geometry: Helicopter
```

Comment:
@bkueng I've added in a section in the actuators doc for this, and we might link to it. Structurally this is odd in actuators because all the other frames split motor and actuator geometry. I made an exception here because there really isn't much to say about helicopter motor geometry.
IN any case, I am thinking of having pages for each frame type because if you're only interested in MC then this page is very intimidating. Further if you are interested in a tiltrotor (say) you end up jumping around for the info. So maybe a bigger restructure to follow. Not here though.

Why have I done this? Because it is good to make sure we fully document the possible values and reasoning for each value. So I've created a framework here and added some questions to help us populate it.